### ISSUE TITLE
Release 1.30.0 not found 404

### ISSUE THREAD

**Issue by alanhamlett** [wakatime/wakatime-cli#594]
@alanhamlett I'm experiencing the same issue and following your instructions above this is the output I get:

```
[WakaTime] Downloading wakatime-cli...
[WakaTime] GitHub API Response 304
[WakaTime] Downloading wakatime-cli from https://github.com/wakatime/wakatime-cli/releases/download/v1.30.0/wakatime-cli-linux-amd64.zip
[WakaTime] Traceback (most recent call last):
  File "vim-wakatime/scripts/install_cli.py", line 194, in downloadCLI
    download(url, zip_file)
  File "vim-wakatime/scripts/install_cli.py", line 454, in download
    urlretrieve(url, filePath)
  File "/usr/lib/python3.9/urllib/request.py", line 239, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
  File "/usr/lib/python3.9/urllib/request.py", line 214, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.9/urllib/request.py", line 523, in open
    response = meth(req, response)
  File "/usr/lib/python3.9/urllib/request.py", line 632, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.9/urllib/request.py", line 561, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found
```

It seems to be trying to download the assets from the `v1.30.0` release which doesn't exist.

The script also keeps deleting the binary if I put it in place manually.

I have the latest commit (02405f1b4077a0bcc99b8111de967582f45ec776) checked out.

I managed to fix the issue by setting `cli_version` to an older version (`v1.28.6`), and `cli_version_last_modified` to an earlier date in `~/.wakatime-internal.cfg`. Not sure how the value of `v1.30.0` got in there in the first place since I've never modified that file manually, but at least it's working now :+1:

_Originally posted by @bausmeier in https://github.com/wakatime/vim-wakatime/issues/109#issuecomment-984612458_

**Comment by alanhamlett** [comment]
@gandarez I think we deleted that release right? Let's do a release to fix this, since GitHub is still reporting 1.30.0 as the latest even thought it's not available. We can't delete releases, only re-release a revert.

**Comment by alanhamlett** [comment]
Fixed by releasing v1.30.1 with #596.

**Comment by gandarez** [comment]
> @gandarez I think we deleted that release right? Let's do a release to fix this, since GitHub is still reporting 1.30.0 as the latest even thought it's not available. We can't delete releases, only re-release a revert.

After deleting a release GH considers the latest correctly. What could happen is the client had some caching which was referencing the wrong one.
